contracts/beyondFramework/dca/libraries/DcaHistoryLib.sol

```diff
@@ -0,0 +1,61 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
```

Comment:
very nice usage of 'using libary' to make calls look like OO, I think we should use it more often too in the 'tokenized' strategies